stanford-sw/translation_maps/library_on_order_map_orig.properties

```diff
@@ -0,0 +1,34 @@
+# Map ON-ORDER Library Codes to User Friendly Description - Long Version for Facet Values
```

Comment:
do we really need to keep both?  Old one is in git.  If we do need both, say something in this comment?